pipelines/release.yml

```diff
@@ -1332,6 +1333,68 @@ jobs:
     params:
       file: version/version
 
+- name: update-images
+  public: true
+  serial: true
+  plan:
+  - in_parallel:
+    - get: periodic-check
+      trigger: true
+    - get: unit-image
+    - get: concourse
+      passed: [shipit]
+    - get: version
+      passed: [shipit]
+    - get: linux-rc
+      passed: [shipit]
+    - get: linux-rc-ubuntu
+      passed: [shipit]
+    - get: concourse-docker
+    - get: builder
+    - get: ci
+  - in_parallel:
+    - task: build-alpine
+      file: concourse-docker/ci/build-image.yml
+      image: builder
+      output_mapping: {image: image-alpine}
+      privileged: true
+    - task: build-ubuntu
+      file: concourse-docker/ci/build-image.yml
+      image: builder
+      input_mapping: {linux-rc: linux-rc-ubuntu}
+      output_mapping: {image: image-ubuntu}
+      privileged: true
+  - in_parallel:
+    - task: watsjs-ubuntu
```

Comment:
rather than a full `watsjs` suite, would the smaller smoke tests suite make sense? (same tests we run in bin-smoke)

Reply:
we did think about that! We chose watsjs because it specifically uses the image passed in. The smoke jobs simply install the binary on a vm and manage it with systemd, not using the image directly at all.

Reply:
so I dug into this a bit more and learned that the bin-smoke job you referred to runs a subset of the watjs smoke tests. Not sure if this is what you meant vs the entire suite. I added a task task that runs only the smoke tests in the watsjs suite.

Reply:
yep that's what i was going for - thanks! just figured it'll save some time since it's really just a sanity check